build.go

```diff
@@ -680,10 +702,77 @@ func Build(opts *BuildArgs) error {
 		}
 	}
 
+	// Revert to original working directory
+	if err := os.Chdir(initialWd); err != nil {
+		return err
+	}
+
 	err = oci.GC(context.Background())
 	if err != nil {
 		fmt.Printf("final OCI GC failed: %v", err)
 	}
 
 	return err
 }
+
+// Build a list of stackerfiles using the same options
+// Note the path in the opts is ignored and changed to by each path in paths
+func BuildMultiple(paths []string, opts *BuildArgs) error {
+
+	// Read all the stacker recipes
+	stackerFiles, err := NewStackerFiles(paths, opts.Substitute)
+	if err != nil {
+		return err
+	}
+
+	// Initialize the DAG
+	dag, err := NewStackerFilesDAG(stackerFiles)
+	if err != nil {
+		return err
+	}
+
+	// In case we don't want to build the images found in includes, filter them out
+	sortedPaths := dag.Sort()
+	var finalPaths []string
+	if !opts.NoIncludes {
+		finalPaths = sortedPaths
+	} else {
+		for _, sp := range sortedPaths {
+			for _, p := range paths {
+				ap, err := filepath.Abs(p)
+				if err != nil {
+					return err
+				}
+				if sp == ap {
+					finalPaths = append(finalPaths, sp)
+				}
+			}
+		}
+	}
+
+	// Show the serial build order
+	fmt.Printf("stacker build order:\n")
```

Comment:
@mikemccracken suggested that we have a separate command to render just the build order. Since we have this logic already, seems like that should be fairly easy?

Reply:
Added a command line flag '--order-only'